src/components/main-page/Registration/HackathonTimer.tsx

```diff
@@ -18,7 +18,7 @@ const datesConfig: ConfigItem[] = [
   { unit: "s", text: "seconds", modulo: 60, className: styles.seconds },
 ];
 
-const hackathonStartDate = dayjs("2022-11-04T17:00:00Z");
+const hackathonStartDate = dayjs("2023-10-21T10:30:00Z");
```

Comment:
Jak robimy refactor to możemy później też zmienić nazwy zmiennych/plików, które się już zdezaktualizowały 😀 Ale ja bym stworzył na to osobnego taska